app/scenes/options_modal/options_modal.js

```diff
@@ -126,6 +123,7 @@ export default class OptionsModal extends PureComponent {
         // By hiding the modal previous to that will cause a rendering bug.
         if (option) {
             option.action();
+            this.props.actions.closeModal();
```

Comment:
Took care of this by adding a closeModal navigation method that will close whatever modal is open. I'm now calling that from within the open action.